.storybook/foundations/corner-rounding/component-size-tokens-medium.mdx

```diff
@@ -0,0 +1,42 @@
+<table>
```

Comment:
It sounds like [there might be a plugin out there](https://github.com/storybookjs/storybook/issues/20091) that would enable us to write the tables in markdown and not HTML, but out of the box our implementation doesn't work with markdown tables.